storage/local/persistence.go

```diff
@@ -879,12 +880,27 @@ func (p *persistence) dropAndPersistChunks(
 	}
 	defer f.Close()
 
+	fi, err := f.Stat()
+	if err != nil {
+		return
+	}
+	totalChunks := int(fi.Size())/chunkLenWithHeader + len(chunks)
+
+	// Calculate chunk index from minShrinkRatio, to skip unnecessary chunk header reading
+	var chunkIndexToStartSeek int
```

Comment:
Write just `chunkIndexToStartSeek := 0` in this case, to be consistent with style in this file.